h/static/scripts/test/streamer-test.js

```diff
@@ -67,26 +73,28 @@ describe('streamer', function () {
   });
 
   it('should send a client ID', function () {
-    assert.equal(socket.messages.length, 1);
-    assert.equal(socket.messages[0].messageType, 'client_id');
-    assert.equal(socket.messages[0].value, streamer.clientId);
+    var fakeWebSocket = socket._socket;
```

Comment:
If we're proxyquiring the `Socket` constructor I'm not sure I see why we need to expose `_socket` like this. Can't we just reference the return value of `new FakeSocket`?